src/useDebouncedCallback.ts

```diff
@@ -55,6 +55,8 @@ function useDebouncedCallback<TCallback extends (...args: any[]) => any>(
 
   const isTimerSetRef = useRef(false)
   const lastArgsRef = useRef<unknown[] | null>(null)
+  // Use any to bypass type issue with setTimeout.
```

Comment:
do we need to fix useTimeout here?

Reply:
The type bypass was actually for the global setTimeout/clearTimeout.

I did notice that these 2 lines didn't use the useTimeout hook, but I didn't change it since I wasn't sure if there was a specific reason for using it:
https://github.com/react-restart/hooks/blob/242ab0bf8c4289f5e0182ed1125ead5d3ab391ad/src/useDebouncedCallback.ts#L164
https://github.com/react-restart/hooks/blob/242ab0bf8c4289f5e0182ed1125ead5d3ab391ad/src/useDebouncedCallback.ts#L171

Can we convert these two calls to use useTimeout?

Reply:
ah sorry you're right. I don't know if I did that intentionally or not, better to leave for now